### DISCUSSION TITLE
CSS not applying on top of HTML

### DISCUSSION THREAD

**Discussion by Mngadinm** [community/community#53884] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Guys,

Im teaching myself HTML & CSS. I have built a project for high school kids. The problem is when i push my files up to github pages , only the HTML skeleton pages without the CSS effects displays. 

here is the website:
https://mngadinm.github.io/valleyacademy.github.io/

Here are my HTML and CSS files: 
https://github.com/Mngadinm/valleyacademy.github.io

Please help , what could be causing this because it works just fine on my local machine,

thanx

**Comment by airtower-luna** [comment]
It looks like [for your recent commits the deployment failed](https://github.com/Mngadinm/valleyacademy.github.io/actions) because the job couldn't get a Jekyll container image. That's a GitHub issue.

I see that the [latest commit with successful deployment](https://github.com/Mngadinm/valleyacademy.github.io/commit/8df9683147ed674b84ac0150ed1c2d9764204dad) is simply missing the `styles.css` file. You've fixed that already, so I'd expect it to work after another deployment attempt.

If deployments keep failing for the same reason, you might want to send a bug report to GitHub support: https://support.github.com/contact/bug-report

**Comment by mohammad-zolghadr** [comment]
Looking at your GitHub repository, I can see that you have a folder named `css` which contains your CSS file named `style.css`. However, when I visit your website and inspect the source code, I can see that the link to your CSS file is incorrect:

```
<link rel="stylesheet" type="text/css" href="/css/styles.css">
```

The problem is that the filename in the link (`styles.css`) does not match the actual filename of your CSS file (`style.css`). You need to change the link to:

```
<link rel="stylesheet" type="text/css" href="/css/style.css">
```

Note that the filename in the link should match the actual filename of your CSS file, including its capitalization.

Once you make this change, your website should display correctly with the CSS styles applied. Be sure to commit and push the changes to your GitHub repository, and then wait a few minutes for the changes to propagate to GitHub Pages.

I hope this helps! Let me know if you have any other questions.

**Comment by emilburganov** [comment]
So css is there, fonts and pictures are not loaded for you. By the way, it's better to use quotes in fontface src
